nominations/views.py

```diff
@@ -278,7 +278,7 @@ def form_valid(self, form):
 
         candidate_name = application.candidate_first_name + ' ' + application.candidate_last_name
         candidate_email = form.cleaned_data['candidate_email']
-        group = application.group
+        group_name = application.group
```

Comment:
Isn't this a local group object? Should be `group_name  = application.group.name` or something like that?

Reply:
👍